app/controllers/api/v1/classifications_controller.rb

```diff
@@ -18,6 +20,13 @@ def update
 
   private
 
+  def filter_by_subject_id
+    if subject_ids = params.delete(:subject_id).try(:split, ',')
```

Comment:
Yeah we should probably do that. I'm going to change that [here](https://github.com/zooniverse/Panoptes/blob/master/app/controllers/concerns/filter_by_owner.rb#L9) too.